examples/yarn-pnp/app/root.tsx

```diff
@@ -0,0 +1,32 @@
+import type { MetaFunction } from "@remix-run/node";
+import {
+  Links,
+  LiveReload,
+  Meta,
+  Outlet,
+  Scripts,
+  ScrollRestoration,
+} from "@remix-run/react";
+
+export const meta: MetaFunction = () => ({
+  charset: "utf-8",
+  title: "New Remix App",
+  viewport: "width=device-width,initial-scale=1",
+});
+
+export default function App() {
+  return (
+    <html lang="en">
+      <head>
+        <Meta />
+        <Links />
+      </head>
+      <body>
+        <Outlet />
+        <ScrollRestoration />
+        <Scripts />
+        <LiveReload port={Number(process.env.REMIX_DEV_SERVER_WS_PORT)} />
```

Comment:
Let's revert this for now, as #3411 still has a discussion going on & we're not sure yet about the solution for the problem.

```suggestion
        <LiveReload />
```